ui/src/app/pages/profile/profile-page.spec.tsx

```diff
@@ -101,6 +101,23 @@ describe('ProfilePageComponent', () => {
 
     getDemographicSurveyButton(wrapper).simulate('click');
     await waitOneTickAndUpdate(wrapper);
+
+    wrapper.find('[data-test-id="checkbox-race-PREFER_NO_ANSWER"]').at(1).simulate('change', {target: {checked: true}})
```

Comment:
I understand this test code is not ideal and can be refactored into better utils. I created ticket [RW-6725](https://precisionmedicineinitiative.atlassian.net/browse/RW-6275) to address this
I am optimizing for time to get this released and would prefer to address issues here in the follow-up.

Reply:
OK works for me